docs/servicenow.md

```diff
@@ -0,0 +1,173 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets is included. Because each
+instance of ServiceNow can have custom fields and custom values, some
+of the tested fields may not be applicable to certain instances of ServiceNow.
+Additionally, your ServiceNow instance may contain ticket fields that we have
+not tested. Keep in mind different ServiceNow tables contain different
+fields with different properties. Mandatory fields in one table may be optional
+or non-existent in another table.
+
+Custom field names and values can be passed in as keyword arguments when
+creating and editing tickets. These are processed by the ServiceNow REST API
+right afterwards. You can also use [get_ticket_content](#content) to get ticket
+content including preset field names.
+
+For more information about REST API and fields see ServiceNow's Wiki:
+- [REST API](http://wiki.servicenow.com/index.php?title=REST_API)
+- [Introduction to Fields](http://wiki.servicenow.com/index.php?title=Introduction_to_Fields)
+- [Dot-Walking](http://wiki.servicenow.com/index.php?title=Dot-Walking)
+
+
+## Table of contents
+- [set_ticket_id()](#set_ticket)
+- [create()](#create)
+- [get_ticket_content()](#content)
+- [edit()](#edit)
+- [add_comment()](#comment)
+- [change_status()](#status)
+- [add_cc()](#add_cc)
+- [rewrite_cc()](#rewrite_cc)
+- [remove_cc()](#remove_cc)
+- [devops_one_url()](#devops_one)
+
+### set_ticket_id(self, ticket_id)
+
+Updates ticket content for the current ticket object. This ticket object is
+assigned using `ticket_id` which is a string.
+
+```python
+# switch to ticket 'ID0123456'
+t.set_ticket_id('ID0123456')
+```
+
+### create(self, short_description, description, category, item, \*\*kwargs) <a name="create"></a>
+
+Creates a ticket. The required parameters for ticket creation are
+short description and description of the issue. Keyword arguments are used
+for other ticket fields.
+
+```python
+t.create(short_description='Ticket summary',
+         description='Ticket description',
+         category='Category',
+         item='ServiceNow')
+```
+
+Tested create() ticket fields:
+
+```python
+category = 'Category'
+item = 'ServiceNow'
+contact_type = 'Email'
+opened_for = 'dept'
```

Comment:
Or did I get it wrong and these are real fields of the ticket created by the code above? Then just change the line "Tested create() ticket fields:" to "Ticket created in one particular ServiceNow instance by the line above has been created with following fields." But this does not make sense to me because of some of the values of the fields below...